yaml/roads.yaml

```diff
@@ -1,5 +1,11 @@
 global:
+  - &output_properties
+    source: {col: source}
```

Comment:
Do we want pistes to have names? It looks like now they don't. They have `piste_name` and `route_name`.